examples/nextjs/.env.local.example

```diff
@@ -1,4 +1,6 @@
 DOTCMS_AUTH_TOKEN=
 NEXT_PUBLIC_DOTCMS_HOST=http://localhost:8080
 NEXT_PUBLIC_EXPERIMENTS_API_KEY=analytic-api-key-from-dotcms-portlet
-NEXT_PUBLIC_EXPERIMENTS_DEBUG=true
\ No newline at end of file
+NEXT_PUBLIC_EXPERIMENTS_DEBUG=true
+# This is for HTTPS server
```

Comment:
Can we add a link with the docs of node for a further explanation of how this works?